docs/extend/index.md

```diff
@@ -97,7 +97,7 @@ started without error.
     ```bash
     $ docker volume ls
 
-    DRIVER              NAME
+    DRIVER              VOLUME NAME
     vieux/sshfs         sshvolume
```

Comment:
ohh :sweat: This is a small regression on master then :sweat: 
/cc @thaJeztah should we put `VOLUME NAME` back ?